src/main/java/uk/gov/hmcts/reform/juddata/camel/binder/JudicialOfficeAppointment.java

```diff
@@ -17,32 +20,39 @@
 public class JudicialOfficeAppointment implements Serializable {
 
     @DataField(pos = 1, columnName = "elinks_id")
+    @NotEmpty
     String elinksId;
 
-    @DataField(pos = 2, columnName = "role_id")
+    @DataField(pos = 2, columnName = "role_id", defaultValue = "0")
```

Comment:
if default value is 0, then if two or more records are having roleid as null then for first record rolled will be 0 and for next record it will fail for constraint violation (duplicate). Is this right ?